ci/e2e-tests-lib.sh

```
@@ -0,0 +1,165 @@
+
+  echo ""
+  if ! git clone --depth 1 --branch "$TARGET_BRANCH" "$E2E_TESTS_REPO" "$E2E_DIR" 2>/dev/null; then
+    echo ">>> Branch '$TARGET_BRANCH' not found in e2e repo, falling back to master"
+    git clone --depth 1 --branch master "$E2E_TESTS_REPO" "$E2E_DIR"
+  else
+    echo ">>> Cloned e2e tests repo (branch: $TARGET_BRANCH) into $E2E_DIR"
+  fi
+
+  echo ">>> Running e2e tests: ELK $ELK_VERSION, image tag: $RUN_TAG"
+  (
+    cd "$E2E_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)